### ISSUE TITLE
Autopost works on Windows but not on Linux (Raspi)

### ISSUE THREAD

**Issue by brain-wash** [ohld/igbot#652]
### Before submitting an issue, make sure you have:
- [x] Updated to the latest version
- [x] Read the [README](https://github.com/instagrambot/instabot/blob/master/README.md) and [docs](https://github.com/instagrambot/instabot/tree/master/docs)
- [x] [Searched](https://github.com/instagrambot/instabot/search?type=Issues) the bugtracker for similar issues including **closed** ones

### Purpose of your issue?
- [x] Bug report (encountered problems/errors)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

I set up autopost and it works just fine when I run it on windows, but whenever I try to run it on my Raspberry Pi it does nothing. The bot logs in and then nothing happens. There is no error message.
I tried it with both, my Raspberry Pi B+ as well as Zero W. Both of them run Raspbian Lite. 

No matter how long I wait, that's all I get:
2018-09-22 13:26:08,916 - INFO - Instabot Started
2018-09-22 13:26:08,935 - INFO - Logged-in successfully as '*******' using the cookie!

**Comment by brain-wash** [comment]
Meanwhile I tried a few things. I installed the full raspbian version (instead of lite) and updated everything, which did not solve the problem.
I tried using the sudo python3 command to run the script (instead of just sudo python), which did not help either.

When I tried debugging the script in the Thonny Python IDE however it is working. What could be the reason that neither python nor python3 are working, but with Thonny it is working?

**Comment by brain-wash** [comment]
I tried so many things in the last few days and I just found the problem. On windows I always went to the folder an ran the script from there. 
On my raspberry pi I always run it like this: sudo python home/pi/....auto_post.py
Either in the command line or by using crontab (I changed the script so it only runs once when executed).
So I did not change the working directory. I would have expected some kind of error message when it can't find the folder for pictures etc, but when I now first go to the correct directory and run the script it seems to be working. Temporarily I use a shell script (to go to the directory and then run the script) until I find a better solution.
